reconciler/errors.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Veroute.on 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconciler
+
+import (
+	"fmt"
+)
+
+// New returns a ReconcilerEvent fully populated.
+func New(eventtype, reason, messageFmt string, args ...interface{}) error {
```

Comment:
`reconciler.New` seems misleading - since I'd expect it to return a reconciler

Reply:
Also why is the file `errors.go`?!

Reply:
oh yeah, sorry I moved this from an error dir in my PoC, will rename.